src/occ/x86/gen.c

```diff
@@ -1,4537 +1,4534 @@
-{
-    OCODE* out = beLocalAlloc(sizeof(OCODE));
-    out->opcode = op_label;
-    out->oper1 = (AMODE*)q->dc.v.label;
-    add_peep(out);
-}
-void asm_goto(QUAD* q) /* unconditional branch */
-{
-    if (q->dc.opcode == i_goto)
-        gen_branch(op_jmp, q->dc.v.label);
-    else /* directbranch */
-    {
-        AMODE* ap;
-        SYMBOL* sp = q->dc.left->offset->v.sp;
-        if (sp->regmode)
-        {
-            ap = makedreg(regmap[q->leftColor & 0xff][0]);
-            ap->liveRegs = q->liveRegs;
-            if (q->dc.left->mode == i_ind)
-            {
-                ap->mode = am_indisp;
-                ap->offset = intNode(en_c_i, 0);
-            }
-        }
-        else
-        {
-            ap = make_offset(q->dc.left->offset);
-            if (q->dc.left->offset->type == en_pc)
-                ap->mode = am_immed;
-            ap->liveRegs = q->liveRegs;
-        }
-        gen_code(op_jmp, ap, 0);
-    }
-}
-void asm_parm(QUAD* q) /* push a parameter*/
-{
-    enum e_op op;
-    AMODE *apl, *aph;
-    getAmodes(q, &op, q->dc.left, &apl, &aph);
-    if (q->dc.left->mode == i_immed)
-    {
-        if (q->dc.left->size >= ISZ_CFLOAT)
-        {
-            int sz = q->dc.left->size, sz1;
-            sz1 = sizeFromISZ(sz);
-            gen_codes(op_sub, ISZ_UINT, makedreg(ESP), aimmed(sz1));
-            pushlevel += sz1;
-            floatLoad(aph, sz - ISZ_CFLOAT + ISZ_FLOAT, FALSE);
-            floatStore(make_stack(-imaginary_offset(sz)), sz - ISZ_CFLOAT + ISZ_FLOAT);
-            pushlevel += sz1;
-            floatLoad(apl, sz - ISZ_CFLOAT + ISZ_FLOAT, FALSE);
-            floatStore(make_stack(0), sz - ISZ_CFLOAT + ISZ_FLOAT);
-        }
-        else if (q->dc.left->size >= ISZ_FLOAT)
-        {
-            int sz = q->dc.left->size, sz1;
-            if (sz >= ISZ_IFLOAT)
-                sz = sz - ISZ_IFLOAT + ISZ_FLOAT;
-            sz1 = sizeFromISZ(sz);
-            floatLoad(apl, sz, FALSE);
-            if (sz1 == 10)
-                sz1 = 12;
-            gen_codes(op_sub, ISZ_UINT, makedreg(ESP), aimmed(sz1));
-            pushlevel += sz1;
-            floatStore(make_stack(0), sz);
-        }
-        else
-        {
-            int sz = q->dc.left->size;
-            if (!isintconst(apl->offset))
-            {
-                gen_codes(op_push, ISZ_UINT, apl, 0);
-                pushlevel += 4;
-            }
-            else if (sz == ISZ_ULONGLONG || sz == -ISZ_ULONGLONG)
-            {
-                gen_codes(op_push, ISZ_UINT, aph, 0);
-                pushlevel += 4;
-                gen_codes(op_push, ISZ_UINT, apl, 0);
-                pushlevel += 4;
-            }
-            else
-            {
-                LLONG_TYPE i;
-                i = apl->offset->v.i;
-                switch (sz)
-                {
-                    case ISZ_USHORT:
-                        i &= 0xffff;
-                        break;
-                    case -ISZ_USHORT:
-               
```

Comment:
These new empty lines are a neat change ;-)

Reply:
Is there a "coding/formatting style" for this repository already? I'd personally prefer an indent definition file + a complete automated change on all source with a (different) PR and keep it out from those changes.

BTW: I like the zpath change and can review this part as "fine".

Reply:
> Interestingly the diff online shows only some new empty lines added while the discussion shows that actually some lines were deleted... [I can't comment on the actual remove of those lines...]

I don't know what git did to this commit, however the main changes here shouldn't have affected anything afaik, I think it may have screwed up what newlines are used, though, or something.